src/common/components/select-repository-list/index.js

```diff
@@ -90,17 +90,18 @@ class SelectRepositoryList extends Component {
     const isLoading = this.props.repositories.isFetching;
     const { selectedRepos } = this.props.selectRepositoriesForm;
     const { repos } = this.props.repositories;
+    const pageLinks = this.renderPageLinks.call(this);
 
     return (
       <div>
-        { this.renderPageLinks.call(this) }
+        { pageLinks }
```

Comment:
I think wireframes only show pagination on the bottom of the list, so it's as good moment as any other to remove the top one.